e2e-tests/constants/routes/insurance/index.js

```diff
@@ -22,7 +22,7 @@ export const INSURANCE_ROUTES = {
     COMPANIES_HOUSE_UNAVAILABLE: `${INSURANCE_ROOT}${ELIGIBILITY_ROOT}/companies-house-unavailable`,
     COMPANY_DETAILS: `${INSURANCE_ROOT}${ELIGIBILITY_ROOT}/company-details`,
     BUYER_COUNTRY: `${INSURANCE_ROOT}${ELIGIBILITY_ROOT}/buyer-location`,
-    INSURED_AMOUNT: `${INSURANCE_ROOT}${ELIGIBILITY_ROOT}/insured-amount`,
+    TOTAL_VALUE_INSURED: `${INSURANCE_ROOT}${ELIGIBILITY_ROOT}/total-value-insured`,
```

Comment:
Can we make sure `INSURED_AMOUNT` no longer exists in the codebase

Also need to update/rename this test: 

```
navigate-to-insured-amount-page-directly.spec.js
```